packages/core/src/interactable-card/InteractableCard.css

```diff
@@ -0,0 +1,136 @@
+  border-color: var(--salt-container-primary-borderColor);
+  --card-accent-color: var(--salt-container-primary-borderColor);
+}
+
+/* Styles applied to InteractableCard when variant="secondary" */
+.saltInteractableCard-secondary {
+  background: var(--saltInteractableCard-background, var(--salt-container-secondary-background));
+  border-color: var(--salt-container-secondary-borderColor);
+  --card-accent-color: var(--salt-container-secondary-borderColor);
+}
+
+/* Styles applied to create accent */
+.saltInteractableCard-accent::after {
+  content: "";
+  position: absolute;
+  background-color: var(--saltInteractableCard-accent-color, var(--card-accent-color));
+}
+
+/* Styles applied to InteractableCard if `accent="bottom"` */
+.saltInteractableCard-accentBottom::after {
+  left: calc(-1 * var(--salt-size-border));
+  bottom: calc(-1 * var(--salt-size-border));
+  height: var(--salt-size-accent);
+  width: calc(100% + calc(2 * var(--salt-size-border)));
+}
+
+/* Styles applied to InteractableCard if `accent="left"` */
+.saltInteractableCard-accentLeft::after {
+  left: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: calc(100% + calc(2 * var(--salt-size-border)));
+  width: var(--salt-size-accent);
+}
+
+/* Styles applied to InteractableCard if `accent="top"` */
+.saltInteractableCard-accentTop::after {
+  left: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: var(--salt-size-accent);
+  width: calc(100% + calc(2 * var(--salt-size-border)));
+}
+
+/* Styles applied to InteractableCard if `accent="right"` */
+.saltInteractableCard-accentRight::after {
+  right: calc(-1 * var(--salt-size-border));
+  top: calc(-1 * var(--salt-size-border));
+  height: calc(100% + calc(2 * var(--salt-size-border)));
+  width: var(--salt-size-accent);
+}
+
+/* Styles applied to InteractableCard on focus */
+.saltInteractableCard:focus-visible {
+  cursor: var(--salt-selectable-cursor-hover);
+  box-shadow: var(--salt-overlayable-shadow-hover);
+  color: var(--saltInteractableCard-color-focus, var(--salt-content-primary-foreground));
+  outline-color: var(--salt-focused-outlineColor);
+  outline-style: var(--salt-focused-outlineStyle);
+  outline-width: var(--salt-focused-outlineWidth);
+  outline-offset: var(--salt-focused-outlineOffset);
+  --card-accent-color: var(--salt-selectable-foreground-hover);
+}
+
+/* Styles applied on hover state to InteractableCard */
+.saltInteractableCard:hover {
+  cursor: var(--salt-selectable-cursor-hover);
+  box-shadow: var(--salt-overlayable-shadow-hover);
+  --card-accent-color: var(--salt-selectable-foreground-hover);
+}
+
+/* Styles applied on active state to InteractableCard and if `selected={true}` */
+.saltInteractableCard-selected,
+.saltInteractableCard:active,
+.saltInteractableCard-active {
+  cursor: var(--salt-selectable-cursor-active);
```

Comment:
nit but i don't know about this one.... the button cursor doesn't change on click but this one does, is there any reason?